### DISCUSSION TITLE
Morpho by weibull distribution

### DISCUSSION THREAD

**Discussion by LifengGan** [neperfepx/neper#782] [Q&A] [answered]
Dear neper developers, 

I am using the -morpho command with Weibull distribution

Refer to the documentation, the `weibull(k,<scale>)`, k represents the shape parameter

By the following command I only got one cell, in version 4.7.1-5
```
neper -T -n from_morpho -domain "square(3,0.5)" -dim 2 -morpho "diameq:weibull(1.3,0.04),circularity:0.95" -o temp
neper -V temp.tess -print img
```
![img](https://github.com/neperfepx/neper/assets/69451718/65d58cfc-8062-4077-b3ee-374921f8c464)

But in the past for version 4.5.1-1, I ran the same command above giving 594 cells
![img](https://github.com/neperfepx/neper/assets/69451718/e5e04b0d-11ca-4db8-a54d-7d81c1e1e801)

Why?
Is there any change in this function for new version?

Best regards,
Lifeng

**Comment by rquey** [comment, marked as answer]
Thanks for the bug report. Fixed in 4.7.1-9.
